ci/scripts/util_checkout.sh

```diff
@@ -19,6 +19,8 @@
 
 # this script is github actions specific to check out the submodules and tags
 
+set -ex
+
```

Comment:
I assume this is intentional to make debugging this job easier in the future but figured I'd double check this is something you want to include.

Reply:
Yes!